tests/udp_socket.rs

```diff
@@ -343,9 +342,15 @@ fn reconnect_udp_socket_receiving() {
         vec![ExpectEvent::new(ID1, Interests::READABLE)],
     );
 
-    // Read only a part of the data.
-    let max = 4;
-    expect_read!(socket1.recv(&mut buf[..max]), &DATA2[..max]);
+    if cfg!(not(target_os = "windows")) {
+        // On all platform except Windows, read only a part of the data.
+        let max = 4;
+        expect_read!(socket1.recv(&mut buf[..max]), &DATA2[..max]);
+    }
+    if cfg!(target_os = "windows") {
+        // On Windows, read all data, reading part of data returns error WSAEMSGSIZE (10040)
+        expect_read!(socket1.recv(&mut buf), DATA2);
+    }
 
     // Now connect back to socket 2, dropping the unread data.
```

Comment:
This needs to be updated if we read the entire packet.

Reply:
In which way needs to be updated?

Reply:
The comment implies data is dropped, but you updated the test to read all the data (no data is dropped). I guess it should say:

```suggestion
    // Now connect back to socket 2.
```